docs/content/operations/api-reference.md

```diff
@@ -0,0 +1,426 @@
+---
+layout: doc_page
+---
+
+# API Reference
+
+This page documents all of the API endpoints for each Druid service type.
+
+## Table of Contents
+  * [Coordinator](#coordinator)
+  * [Overlord](#overlord)
+  * [MiddleManager](#middlemanager)
+  * [Peon](#peon)
+  * [Broker](#broker)
+  * [Historical](#historical)
+
+## Common
+
+The following endpoints are supported by all nodes.
+
+### GET
+
+* `/status`
+
+Returns the Druid version, loaded extensions, memory used, total memory and other useful information about the node.
+
+* `/health`
```

Comment:
This should actually be `/status/health`

Reply:
fixed